src/__tests__/components/frontend-engine/yup/custom-conditions.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { YupHelper } from "src/components/frontend-engine/yup";
+import { TestHelper } from "src/utils";
+
+describe("validateUinfin", () => {
```

Comment:
note: test cases are similar to mol-lib-api-contract because the uinfin check is copied from there